src/objects/entities/Ben3.ts

```diff
@@ -119,12 +119,39 @@ class Ben3 extends Entity {
     }
   }
 
+  left() {
+    const rot = this.gameObject.rotation - Phaser.Math.DegToRad(5);
+    this.gameObject.setRotation(rot);
+    this.head.body.mass = 0;
+    // this.gameObject.setAngle(this.gameObject.angle - 5);
+  }
+
+  right() {
+    const rot = this.gameObject.rotation + Phaser.Math.DegToRad(5);
+    this.gameObject.setRotation(rot);
+    // this.head.body.mass = 0;
+    // this.gameObject.setAngle(this.gameObject.angle + 5);
+  }
```

Comment:
i had a go of this branch, using A and D to rotate torso...
it seems to me it's not working when the head gets big because the rotation is only applied to "body" part of ben... 
i feel like it needs to become a higher force, proportional to head mass as there is more weight to lift when ben has big head

:thinking:  a lot to think about here, contact me if you want to riff on ideas

Reply:
ignore my comment most of what i said is irrelevant

Reply:
i feel like using Phaser's `setRotation` on Matter body, is like telling the matter engine, that we know better than it does... effectively overriding its physics calculations

the keepUpright we had before was using Matter's `setAngularVelocity` - which is still an override, just an override of angular velocity, rather than rotational position. but uses matter's functions and therefore harmonizes more with Matter engine

a step below this is to apply forces, not sure if that can be done to rotation... this does not override any engine values